app/(route)/playground/[uid]/_nav.tsx

```
@@ -0,0 +1,244 @@
+  return (
+    <div className="fixed top-[123px] left-[50%] -translate-x-1/2 w-[450px] h-[350px] bg-white/70 rounded-[5px] text-center text-black text-[35px] grid grid-cols-4 shadow-[10px_10px_10px_0px_rgba(0,0,0,0.25)]">
+      {Array.from({ length: 16 }, (_, index) => {
+        const pieceNumber = (index + 2) * (index + 2);
+        return (
+          <div
+            key={index}
+            className={cx('flex items-center justify-center rounded-[5px] border-[7px] border-transparent text-center text-[35px] hover:border-white', {
+              'bg-white': selectedPieceNumber === pieceNumber,
+            })}
+            onClick={() => handleClickPieceNumber(pieceNumber)}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)